static/js/components/dashboard/util.js

```diff
@@ -16,3 +17,26 @@ export const courseListToolTip = (text: string, id: string) => (
     </ReactTooltip>
   </div>
 );
+
+export const isCurrentlyEnrollable = (enrollmentStartDate, now) => (
+  enrollmentStartDate !== null &&
+    enrollmentStartDate !== undefined &&
+    enrollmentStartDate.isSameOrBefore(now || moment(), 'day')
+);
+
+export const formatGrade = (grade) => {
+  if (_.isNil(grade) || grade === '') {
+    return '';
+  } else {
+    let gradeString;
+    if (_.isString(grade)) {
+      gradeString = _.round(Number(grade));
+    } else if (_.isInteger(grade)) {
+      gradeString = grade;
+    } else if (_.isNumber(grade)) {
+      // If 'grade' is a number but not an integer, round it.
+      gradeString = _.round(grade);
+    }
+    return gradeString ? `${gradeString}%` : '';
```

Comment:
This would fail if `gradeString` is `0`. It seems like `${gradeString}%` should cover all cases here since the code checks for nil above